ubuntuStation/models.py

```diff
@@ -1,6 +1,6 @@
 from . import db
 from flask_login import UserMixin
-from sqlalchemy import PrimaryKeyConstraint
+from sqlalchemy import PrimaryKeyConstraint, desc
```

Comment:
desc is basically an `ORDER BY DESC` for sqlalchemy querries. See [docs](http://docs.sqlalchemy.org/en/latest/core/sqlelement.html#sqlalchemy.sql.expression.desc) for more explanation. This has nothing to do with keys. The reason why this is added was so that when displaying `deployed_urls`, I can display to the user all ACTIVE urls first, then DISABLED. If not, they would be mixed together and it ended up just looking cleaner for the user to see all active, then all disabled. Minor detail.